website/orders/api/v1/views.py

```diff
@@ -76,9 +77,9 @@ def post(self, request, **kwargs):
         except ValueError:
             return Response(status=status.HTTP_400_BAD_REQUEST)
         try:
-            place_orders(cart.get_item_list(), request.user, shift)
+            add_user_orders(cart.get_item_list(), shift, request.user)
         except OrderException as e:
-            raise ValidationError(e.__str__())
```

Comment:
I think we should actually raise an exception, but maybe a `PermissionDenied`. DRF will catch them right?

Reply:
Good point, I'll change them all (it will work the same as DRF will just return a 403).